src/index.html

```diff
@@ -7,8 +7,198 @@
+              <div id="expr_T3" class="expr_T"></div>
+              <div id="expr_time3" class="expr_time"></div>
+              <div id="expr_P3" class="expr_P"></div>
+            </div>
+          </li>
+          <li>
+            <div id="educ_section">
+              <div id="educ_t">EDUCATION</div>
+              <div id="educ_down">DOWNLOAD MY FULL RESUME</div>
+              <hr />
+              <div id="educ_T1" class="educ_T"></div>
+              <div id="educ_time1" class="educ_time"></div>
+              <div id="educ_P1" class="educ_P"></div>
+              <div id="educ_T2" class="educ_T"></div>
+              <div id="educ_time2" class="educ_time"></div>
+              <div id="educ_P2" class="educ_P"></div>
+              <div id="educ_T3" class="educ_T"></div>
+              <div id="educ_time3" class="educ_time"></div>
+              <div id="educ_P3" class="educ_P"></div>
+            </div>
+          </li>
+          <li>
+            <div id="hobb_section">
+              <div id="hobb_t">HOBBIES&INTERESTS</div>
+              <div id="hobb_down">DOWNLOAD MY FULL RESUME</div>
+              <hr />
+              <img src="" alt="hobby_1" class="hobb_pic" />
+              <img src="" alt="hobby_2" class="hobb_pic" />
+              <img src="" alt="hobby_3" class="hobb_pic" />
+              <img src="" alt="hobby_4" class="hobb_pic" />
+              <img src="" alt="hobby_5" class="hobb_pic" />
+              <img src="" alt="hobby_6" class="hobb_pic" />
+            </div>
+          </li>
+        </ul>
+      </div>
+
+      <div id="skls">
+        <div id="skls_t">SKILLS</div>
+        <hr />
+        <div id="skls_map">
+          <div id="skls_1" class="skls_total">
+            <div id="skilname_1" class="skilname">Microsoft</div>
+            <div id="skilprof_1" class="skilprof"></div>
+          </div>
+          <div id="skls_2" class="skls_total">
+            <div id="skilname_2" class="skilname">Microsoft</div>
+            <div id="skilprof_2" class="skilprof"></div>
+          </div>
+          <div id="skls_3" class="skls_total">
+            <div id="skilname_3" class="skilname">Microsoft</div>
+            <div id="skilprof_3" class="skilprof"></div>
+          </div>
+          <div id="skls_4" class="skls_total">
+            <div id="skilname_4" class="skilname">Microsoft</div>
+            <div id="skilprof_4" class="skilprof"></div>
+          </div>
+          <div id="skls_5" class="skls_total">
+            <div id="skilname_5" class="skilname">Microsoft</div>
+            <div id="skilprof_5" class="skilprof"></div>
+          </div>
+          <div id="skls_6" class="skls_total">
+            <div id="skilname_6" class="skilname">Microsoft</div>
+            <div id="skilprof_6" class="skilprof"></div>
+          </div>
+        </div>
+      </div>
+      <div id="conta">
+        <div id="quicid">
+          <div id="quicid_t">QUICK ID</div>
+          <hr />
```

Comment:
In semantic perspective, `<hr>` means a thematic change, so I suggest here it's better to use `<div>` then style it 
https://www.w3docs.com/learn-html/html-hr-tag.html